src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Deployment/CreateOrderStatuses.sql

```diff
@@ -1,10 +1,11 @@
-﻿DECLARE @submitted AS nvarchar(30) = N'Submitted';
-DECLARE @unsubmitted AS nvarchar(30) = N'Unsubmitted';
+﻿DECLARE @complete AS nvarchar(30) = N'Complete';
+DECLARE @incomplete AS nvarchar(30) = N'Incomplete';
 
 IF NOT EXISTS (
   SELECT *
   FROM dbo.OrderStatus
-  WHERE [Name] IN (@submitted, @unsubmitted))
+  WHERE [Name] IN (@complete, @incomplete))
 BEGIN
-	INSERT INTO dbo.OrderStatus ([Name]) VALUES (@submitted), (@unsubmitted)
+	INSERT INTO dbo.OrderStatus (OrderStatusId, [Name]) VALUES (1, @complete);
+    INSERT INTO dbo.OrderStatus (OrderStatusId, [Name]) VALUES (2, @incomplete);

```

Comment:
This could (and should, in my opinion) be a single `INSERT` statement.

I think there's also a mix of tabs and spaces here.

A bit of an aside, as I can see this is just an edit, but is there any reason why all the other post-deployment files (other than `CreateTables.sql` although I'm not sure why we still have that)  follow the format `Insert[Thing]` but this one is `Create[Thing]`?

Reply:
I can fix those 👍